Civi/Install/Requirements.php

```diff
@@ -227,8 +227,7 @@ public function checkMysqlConnection(array $db_config) {
       'details' => "Connected",
     );
 
-    $host = isset($db_config['server']) ? $db_config['server'] : $db_config['host'];
-    $conn = @mysqli_connect($host, $db_config['username'], $db_config['password']);
+    $conn = @mysqli_connect($host, $db_config['username'], $db_config['password'], $db_config['database'], $db_config['port']);
```

Comment:
I think we should not use $db_config['server'] because $db_config['server'] is set to the following : 

'server' => "{$civicrm_db['host']}:{$civicrm_db['port']}",  (modules/civicrm/civicrm.install)

Hence it would result in the same problem as earlier (host:port issue of mysqli_connect). This leaves us with only one line to make connection. Do we still need to make it a different function?

Let me know, I will make it as you suggest. I apologies for the wrong checkin ($host). That must have been taken care of.

Regards,
Vasantha Kaje.